packages/svelte/tests/validator/samples/document-binding-invalid-dimensions/errors.json

```diff
@@ -1,7 +1,7 @@
 [
 	{
 		"code": "bind_invalid_name",
-		"message": "`bind:clientWidth` is not a valid binding. Possible bindings for <svelte:document> are focused, fullscreenElement, visibilityState, this",
+		"message": "`bind:clientWidth` is not a valid binding. Possible bindings for <svelte:document> are focused, activeElement, fullscreenElement, pointerLockElement, visibilityState, this",
```

Comment:
not strictly related to this PR, but a note to self: what is `focused`? `document.focused` isn't a thing so this seems odd, maybe a bug? also it'd be nice if these things were sorted alphabetically rather than randomly. will investigate after lunch

Reply:
okay, looks like we added `focused` to all elements a while back. it should probably exclude `<svelte:document>` and some other things. can look into that as a separate PR